cps/templates/author.html

```diff
@@ -0,0 +1,65 @@
+{% extends "layout.html" %}
+{% block body %}
+<h2>{{title}}</h2>
+
+{% if author is not none %}
+<section class="author-bio">
+  {%if author['image_url'] is not none %}
+  <img src="{{author.image_url}}" alt="{{author.name}}" class="author-photo pull-left">
```

Comment:
I'm not able to recreate the issue. I downloaded a book by Honoré de Balzac, added it to my library, and without any changes the page loads without an error.